workflow-config/config-android/src/main/java/com/squareup/workflow1/config/AndroidRuntimeConfigTools.kt

```diff
@@ -26,6 +29,7 @@ public class AndroidRuntimeConfigTools {
     @WorkflowExperimentalRuntime
     public fun getAppWorkflowRuntimeConfig(): RuntimeConfig {
       return when (BuildConfig.WORKFLOW_RUNTIME) {
+        "stateChange" -> RenderOnStateChangeOnly
         "conflate" -> ConflateStaleRenderings
```

Comment:
So we can't turn on both `RenderOnStateChangeOnly` and `ConflateStaleRenderings`? Isn't that going to bite us pretty quickly?

Reply:
Combined now to make both usable at the same time (and to test that).